packages/components/src/components/autocomplete/autocomplete.tsx

```diff
@@ -61,26 +61,26 @@ const getFilteredItems = (val: string, list: TItem[], should_filter_by_char = fa
 const Autocomplete = React.memo((props: TAutocompleteProps) => {
     const {
         autoComplete,
-        data_testid,
```

Comment:
Please add a QA checklist to make sure all the instances of this component throughout the code is being tested.